packages/clis/parser/game-files/def-parser.ts

```diff
@@ -664,34 +721,114 @@ function processAchievementsJson(
     });
   }
 
+  //
+  // achievementEachDeliveryPoint
+  //
+  for (const a of Object.values(obj.achievementEachDeliveryPoint)) {
+    achievements.set(a.achievementName, {
+      type: 'eachDeliveryPoint',
+      sources: a.sources,
+      targets: a.targets,
+    });
+  }
+
+  //
+  // achievementOversizeRoutesData
+  //
+  for (const a of Object.values(obj.achievementOversizeRoutesData)) {
+    achievements.set(a.achievementName, {
+      type: 'oversizeRoutesData',
+    });
+  }
+
+  return achievements;
+}
+
+function processAtsAchievementsJson(
+  obj: AtsAchievementsSii,
+): Map<string, Achievement> {
+  const achievements = processBaseAchievementsJson(obj);
+
   //
   // achievementFerryData
   //
   for (const a of Object.values(obj.achievementFerryData)) {
     achievements.set(a.achievementName, {
       type: 'ferryData',
-      endpointA: a.endpointA ?? 'TODO',
-      endpointB: a.endpointB ?? 'TODO',
+      endpointA: a.endpointA,
+      endpointB: a.endpointB,
     });
   }
 
+  return achievements;
+}
+
+function processEts2AchievementsJson(
+  obj: Ets2AchievementsSii,
+): Map<string, Achievement> {
+  const achievements = processBaseAchievementsJson(obj);
+
   //
-  // achievementEachDeliveryPoint
+  // achievementFerryData
   //
-  for (const a of Object.values(obj.achievementEachDeliveryPoint)) {
+  for (const a of Object.values(obj.achievementFerryData)) {
     achievements.set(a.achievementName, {
-      type: 'eachDeliveryPoint',
-      sources: a.sources,
-      targets: a.targets,
+      type: 'ferryDataByType',
+      ferryType: a.ferryType,
```

Comment:
ETS2 has ferry achievements that just require riding any ferry or a train (vs. ATS ferry achievement, which requires riding a specific ferry line).